pages/k8s/install-offline.md

```diff
@@ -212,28 +212,31 @@ from an internet connected machine:
 ```bash
 git clone https://github.com/charmed-kubernetes/cdk-shrinkwrap.git /tmp/.shrinkwrap
 cd /tmp/.shrinkwrap
-BUNDLE=cs:charmed-kubernetes-862       # Choose a deployment bundle (example is 1.23.x)
+BUNDLE='charmed-kubernetes --channel=1.26/stable'       # Choose a deployment bundle (example is 1.26.x)
 ./shrinkwrap-lxc.sh $BUNDLE 
 ls /tmp/.shrinkwrap/build/
 ```
 
-In air-gapped environment with access to the Juju controller, 
+In the air-gapped environment with access to the Juju controller:
 1. Extract the tar.gz file
 1. Print Available instructions from the deploy.sh
    1. Push the snaps to the snap-store-proxy
    1. Push the container images to your offline container registry
 1. Ensure the Juju environment is configured to pull from the snap-store-proxy and container registry
    1. This will require configuration changes on the `containerd` application and `kubernetes-control-plane` application in the `./bundle.yaml`
    1. Ensure `applications.containerd.options` includes `custom_registries` settings
-1. Finally, deploy the Juju charms and resources from the provided local bundle.
-```bash
-tar -xvf cs:charmed-kubernetes-862-stable-*.tar.gz --force-local
-cd cs:charmed-kubernetes-862-stable-*/
-./deploy.sh
-# examine provided instructions
-# ensure necessary modifications are considered
-juju deploy ...
-```
+1. Finally, deploy the Juju charms and resources from the provided local bundle:
+
+   ```bash
+   tar -xvf charmed-kubernetes-*.tar.gz --force-local
```

Comment:
i *think* this works but will be able to check the filename when i have finished downloading the internet